// getting errors

const x = () => {
    const a = 10
    return a
}

console.log('value of function', x())

function add(x) {
const b = 10
console.log(x)
return x + b
}

console.log(add())